go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -17,7 +17,7 @@ class GoModUpdater
         RESOLVABILITY_ERROR_REGEXES = [
           # The checksum in go.sum does not match the downloaded content
           /verifying .*: checksum mismatch/.freeze,
-          /go: .*: go.mod has post-v\d+ module path/
+          /go get: .*: go.mod has post-v\d+ module path/
```

Comment:
Do we only care about the case where `go get` specifically raises this error? It seems OK to be more lenient when matching on the error?

Reply:
Agree! It should match whether emitted by `go mod` tooling or `go get` tooling.

I assume this would match both in ruby? But still tight enough to only match _either_ `go:` or `go get:` and nothing else?

```suggestion
          /go (?:get)?: .*: go.mod has post-v\d+ module path/
```

Reply:
Yeah that'd work, or maybe even just this?:

```ruby
/go *. go.mod has post-v\d+ module path/
```

Reply:
For now, I'd probably lean toward the slightly more restrictive regex, as `*.` has too often bitten me unexpectedly. It's easy enough to relax it later, but this at least communicates to the reader/dev what the expected matching is.